.changes/unreleased/BUG FIXES-20251001-152300.yaml

```diff
@@ -0,0 +1,5 @@
+kind: BUG FIXES
+body: 'fwserver: update validation list result validation to check if an identity''s values are all null'
```

Comment:
nit: typo
```suggestion
body: 'fwserver: update validation list result validation to check if an identity's values are all null'
```

Reply:
Actually, ignore this suggestion. The extra `'` is acting as an escape character 😅